### PR TITLE
Added mail verification

### PR DESCRIPTION
**Pull request by e7hdk**
I reconfigured mail verification, now It is running successfully

### CODE REVIEW

File: project/backend/app/app/settings.py [hunk 1]
```diff
@@ -84,11 +84,11 @@
...
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'group5db_451',
+        'NAME': 'postgres',
         'USER': 'postgres',
-        'PASSWORD': 'group5',
+        'PASSWORD': 'password',
         'HOST': 'localhost',
-        'PORT': '',
+        'PORT': '5432',
```
Comment (GozdeUnver) [inline project/backend/app/app/settings.py:91]: Could you restore the changes here and make them the same as they are in master

Comment (e7hdk) [reply project/backend/app/app/settings.py]: I restored the changes

File: project/backend/app/register/views.py [hunk 2]
```diff
@@ -75,20 +85,27 @@ def register(request):
...
 
-        interest1 = InterestLevel.objects.create(owner_of_interest=user, sport_name=sport1, skill_level=level1)
-        interest2 = InterestLevel.objects.create(owner_of_interest=user, sport_name=sport2, skill_level=level2)
+        #interest1 = InterestLevel.objects.create(owner_of_interest=user, sport_name=sport1, skill_level=level1)
+        #interest2 = InterestLevel.objects.create(owner_of_interest=user, sport_name=sport2, skill_level=level2)
 
-        interest1.save()
-        interest2.save()
+        #interest1.save()
+        #interest2.save()
```
Comment (GozdeUnver) [inline project/backend/app/register/views.py:108]: why are you not saving the skill level and sport interests while registering?

Comment (GozdeUnver) [reply project/backend/app/register/views.py:108]: you should use Sport and SkillLevel models in the post app

Comment (e7hdk) [reply project/backend/app/register/views.py:108]: There were no example sports on my localhost. I will change

Comment (e7hdk) [reply project/backend/app/register/views.py]: I fixed that problem

**Comment by GozdeUnver** [comment]
In your test files, you should define sports like this: "Sport.objects.create(id=1,sport_name="football",is_custom=False)" because we added a new field called is_custom so it gives error when you don't add it. Secondly, in your test_existing_mail and test_existing_username methods, you are not givin a username of length bigger than 8 so your register function returns 401 instead of 409 so your tests fail. When these modifications are done you can merge it with master

**Comment by e7hdk** [comment]
> 

I did what you ask

**Pull request merged**